data/human/free worlds 0 prologue.txt

```diff
@@ -1711,6 +1711,19 @@ mission "FW Katya 4"
 			
 			`	"Our fight isn't with the Navy," says Katya. "Most of them are good people. I'm sure they don't want war any more than we do. They're like police officers who are suddenly being asked to fight against the very people they swore to protect and serve."`
 			`	"I was a child on Vinci while this was happening," says Eyes. "A Paradise World. I would listen to the news and could never understand why all the adults around me were so very, very angry that the Navy had gone off to rescue two million people from freezing to death. But they told me that we lived in one little island of safety in a dangerous galaxy, and that without the Navy protecting us the pirates or even maybe aliens would sweep in and destroy our homes and take our money. That it was us against the whole galaxy."`
+			branch paradise
+				not "start: paradise"
+			
+			`	You talk to Eyes about your own upbringing on Mainsail, how you always dreamed of taking to the stars instead of your family's business, and how your father gave you a pilot's license for your eighteenth birthday.`
+			`	"It sounds like you've come a long way from the Paradise Worlds too," says Eyes. "If you don't mind me asking, what do you think your father would think about you helping the Free Worlds?"`
+			choice
```

Comment:
I think it'd be nice if at least _some_ of these options varied Eyes' response. Can still be small, maybe something like `Eyes averts his gaze` for the "rather not say" choice, or `Eyes nods, smiling` if FL's father would have been happy.